src/kernels/jupyter/launcher/notebookServerProvider.ts

```diff
@@ -176,13 +180,15 @@ export class NotebookServerProvider implements IJupyterServerProvider {
             if (activeInterpreter) {
                 const displayName = activeInterpreter.displayName
                     ? activeInterpreter.displayName
-                    : activeInterpreter.uri.fsPath;
+                    : getFilePath(activeInterpreter.uri);
                 throw new Error(
                     DataScience.jupyterNotSupportedBecauseOfEnvironment().format(displayName, e.toString())
                 );
             } else {
                 throw new JupyterInstallError(
-                    DataScience.jupyterNotSupported().format(await this.jupyterExecution.getNotebookError())
+                    DataScience.jupyterNotSupported().format(
+                        this.jupyterExecution ? await this.jupyterExecution.getNotebookError() : 'Error'
```

Comment:
Minor, but I'm pretty sure this is user visible so the 'Error' needs loc.

Reply:
This case will never happen. It's just for the compiler. This is for 'starting' jupyter which we'll never do in web.